distsql/xeval/eval.go

```diff
@@ -54,298 +48,32 @@ type decodedValueList struct {
 // Eval evaluates expr to a Datum.
 func (e *Evaluator) Eval(expr *tipb.Expr) (types.Datum, error) {
 	switch expr.GetTp() {
-	case tipb.ExprType_Null:
-		return types.Datum{}, nil
-	case tipb.ExprType_Int64:
-		return e.evalInt(expr.Val)
-	case tipb.ExprType_Uint64:
-		return e.evalUint(expr.Val)
-	case tipb.ExprType_String:
-		return e.evalString(expr.Val)
-	case tipb.ExprType_Bytes:
-		return types.NewBytesDatum(expr.Val), nil
-	case tipb.ExprType_Float32:
-		return e.evalFloat(expr.Val, true)
-	case tipb.ExprType_Float64:
-		return e.evalFloat(expr.Val, false)
-	case tipb.ExprType_MysqlDecimal:
-		return e.evalDecimal(expr.Val)
-	case tipb.ExprType_MysqlDuration:
-		return e.evalDuration(expr.Val)
-	case tipb.ExprType_ColumnRef:
-		return e.evalColumnRef(expr.Val)
-	case tipb.ExprType_LT:
-		return e.evalLT(expr)
-	case tipb.ExprType_LE:
-		return e.evalLE(expr)
-	case tipb.ExprType_EQ:
-		return e.evalEQ(expr)
-	case tipb.ExprType_NE:
-		return e.evalNE(expr)
-	case tipb.ExprType_GE:
-		return e.evalGE(expr)
-	case tipb.ExprType_GT:
-		return e.evalGT(expr)
-	case tipb.ExprType_NullEQ:
-		return e.evalNullEQ(expr)
-	case tipb.ExprType_And:
-		return e.evalAnd(expr)
-	case tipb.ExprType_Or:
-		return e.evalOr(expr)
-	case tipb.ExprType_Like:
-		return e.evalLike(expr)
-	case tipb.ExprType_Not:
-		return e.evalNot(expr)
-	case tipb.ExprType_In:
-		return e.evalIn(expr)
+	// data type.
```

Comment:
s/data/Data or remove `.`
The same as line57, line62, line65, line69 and the same as some comments in  plan_test.go